base/applications/mspaint/history.cpp

```diff
@@ -209,6 +209,45 @@ void ImageModel::InvertColors()
     NotifyImageChanged();
 }
 
+void ImageModel::GSColors()
```

Comment:
This function, as implemented right now, seems to merely imitate the cosmetic effects of a pixel format conversion.
A grayscale image should use 8 bits per pixel, as opposed to a true color image with 24 bits per pixel.

Reply:
Yes, I agree. It's a workaround better than leave unimplemented function.

Reply:
Maybe leave grayscale support for later, entirely. See below.